services/ingestion/engine.go

```diff
@@ -217,6 +220,16 @@ func (e *Engine) processEvents(events *models.CadenceEvents) error {
 
 // indexEvents will replay the evm transactions using the block events and index all results.
 func (e *Engine) indexEvents(events *models.CadenceEvents, batch *pebbleDB.Batch) error {
+	if events.FeeParameters() != nil {
+		if err := e.feeParameters.Store(events.FeeParameters(), batch); err != nil {
```

Comment:
Let's log when this happens. I believe this event shouldn't happen very often, so it's maybe worth log with both the old value and new value .